archlinuxcn/serenity/PKGBUILD

```diff
@@ -0,0 +1,54 @@
+# Maintainer:  Misaka13514 <Misaka13514 at gmail dot com>
+# Contributor: Henry-ZHR <henry-zhr@qq.com>
+
+pkgname=serenity
+pkgver=1.0.0_beta.13
+_pkgver=${pkgver//_/-}
+pkgrel=2
+pkgdesc='The configuration generator for sing-box'
+arch=('i686' 'x86_64' 'aarch64' 'armv7h')
+url='https://github.com/SagerNet/serenity'
+license=('custom:GPL-3.0-or-later WITH name use or association addition')
```

Comment:
The license file should be installed to `/usr/share/licenses/serenity/LICENSE` since it's a custom license.